controllers/report_controller.go

```diff
@@ -54,7 +59,8 @@ func (r *ReportReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 	l.Info("reconciling report", "req", req.NamespacedName)
 	defer l.Info("finished reconciling report", "req", req.NamespacedName)
 
-	report := &curatorv1alpha1.Report{}
+	reportOriginal := &curatorv1alpha1.Report{}
+	report := reportOriginal.DeepCopy()
```

Comment:
nit: I don't think we need to create a deepcopy as controller-runtime should be created a deepcopied version from the informer cache for us anyways. The Report resource we're grabbing the cache shouldn't be read-only, so we can make modifications to that object directly.